src/nodedef.cpp

```diff
@@ -1442,6 +1445,9 @@ void ContentFeatures::deSerializeOld(std::istream &is, int version)
 		liquid_alternative_source = deSerializeString(is);
 		liquid_viscosity = readU8(is);
 		light_source = readU8(is);
+		if (light_source > LIGHT_MAX) {
+			light_source = LIGHT_MAX;
+		}
```

Comment:
Braces *can* be removed but that doesn't mean they need to be.

Our code style supports either.  The only thing we ask is that an author stays consistent with the rest of the surrounding code.